app/code/Magento/GroupedProduct/Model/Quote/Item/GroupedItemQty.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\GroupedProduct\Model\Quote\Item;
+
+use Magento\Framework\Model\AbstractExtensibleModel;
+use Magento\GroupedProduct\Api\Data\GroupedItemQtyInterface;
+use Magento\GroupedProduct\Api\Data\GroupedItemQtyExtensionInterface;
+
+/**
+ * Object that contains quantity information for a single associated product of a Grouped Product
+ *
+ * Class \Magento\GroupedProduct\Model\Quote\Item\GroupedItemQty
+ */
+class GroupedItemQty extends AbstractExtensibleModel implements GroupedItemQtyInterface
```

Comment:
When the interface will be refactored, please, remove the inheritance from the `AbstractExtensibleModel`, we don't need a heavy model for the data object.